codespell_lib/data/dictionary.txt

```diff
@@ -5693,6 +5713,7 @@ calulation->calculation
 calulations->calculations
 caluse->clause, callus, callous,
 Cambrige->Cambridge
+camelion->chameleon
```

Comment:
`cameleon` would be a nice addition, in this PR or different one (especially for French speakers).

Reply:
Added.